external-import/misp/src/misp.py

```diff
@@ -1316,16 +1316,18 @@ def prepare_elements(self, galaxies, tags, author, markings):
                     else:
                         aliases = [name]
                     if name not in added_names:
+                        x_mitre_id = None
+                        if "external_id" in galaxy_entity["meta"]:
+                            if len(galaxy_entity["meta"]["external_id"]) > 0:
```

Comment:
One thing to pen in is what we do when `external_id` has >1 items. It looks like the UI supports 1 `x_mitre_id`